src/pages/Social.jsx

```diff
@@ -4,82 +4,107 @@ import BlueDiscord from '../assets/blueDiscord.png';
-                <img src={FusionFriday} alt={`Fusion Friday`} className="w-307 h-307 object-cover flex-shrink-0 mb-10"/>
-                <img src={FusionFriday} alt={`Fusion Friday`} className="w-307 h-307 object-cover flex-shrink-0 mt-10"/>
-                <img src={FusionFriday} alt={`Fusion Friday`} className="w-307 h-307 object-cover flex-shrink-0 mb-10"/>
-                <img src={FusionFriday} alt={`Fusion Friday`} className="w-307 h-307 object-cover flex-shrink-0 mt-10"/>
-            </div>
-            <div className="max-w-[1088px] w-full flex flex-col justify-center items-center gap-10 mt-60">
-                <h1 className="text-black text-center font-inter text-4xl font-medium" style={{ letterSpacing: '8px' }}>
-                    FTEK
-                </h1>
-                <p className="text-black text-center font-inter text-base leading-7 px-36">
-                    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et 
-                    dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip 
-                    ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu 
-                    fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt 
-                    mollit anim id est laborum.
-                </p>         
-            </div>
-            <div className="flex flex-row mt-40 gap-20 overflow-hidden justify-center">
-                <img src={Ftek} alt={`Ftek`} className="w-857 h-510 object-cover flex-shrink-0 mt-10"/>
-                <img src={Ftek} alt={`Ftek`} className="w-857 h-510 object-cover flex-shrink-0 mt-10"/>
-                <img src={Ftek} alt={`Ftek`} className="w-857 h-510 object-cover flex-shrink-0 mt-10"/>
-            </div>
-            <div className="max-w-[1088px] w-full flex flex-col justify-center items-center gap-10 mt-60">
-                <h1 className="text-black text-center font-inter text-4xl font-medium" style={{ letterSpacing: '8px' }}>
-                    GAMING
-                </h1>
+
+            <div className="w-full bg-[#F4F4F4] flex mt-40 overflow-x-hidden">
+                <div className="max-w-[1088px] w-full flex flex-col gap-10 relative">
+                    <img src={WhiteHalfSun} alt={`White Half Sun`} className="w-1211.6 h-964.12 left-0 object-cover flex-shrink-0 z-0"/>
+                    
+                    <img src={FusionFridays} alt={`Fusion Friday Karaoke`} className="w-1381 h-576 left-308 object-cover flex-shrink-0 absolute z-10" style={{top: '300px', left: '308px' }}/>
+                    <div className="w-[637px] h-[514px] p-[44px_0_0_0] gap-[32px] rounded-[72px_72px_72px_0] bg-white flex-shrink-0 absolute z-10" style={{top: '200px', left: '1060px' }}> 
```

Comment:
try using the width values that are provided in the tailwind docs ^ , i know these are the lengths that are provided in the figma, but they are only set for one specific screen size while there's a lot of variation in screen sizes that exist